.github/workflows/CI.yml

```diff
@@ -157,21 +159,21 @@ jobs:
 
       # Upload Regression Results as Dynamic if Push to non-main Branch
       - name: Upload Results as Dynamic
-        if: github.ref != 'refs/heads/main'
+        if: !( env.REFERENCE_JOB )
```

Comment:
This change will make it only upload results as dynamic if it's pushed to `ReactionMechanismGenerator/RMG-Py`, but previously it should upload for a forked branch as well. I think we should allow upload as dynamic for forked ones to encourage people to develop with forked repos and keep the branches on `ReactionMechanismGenerator/RMG-Py` clean.

Reply:
I think the opposite. This change _increases_ the number of things that are now uploaded as dynamic. Previously, `main` on any fork would upload as its own "reference" job. Because we define above `REFERENCE_JOB: ${{ github.ref == 'refs/heads/main' && matrix.os =='ubuntu-latest' && github.repository == 'ReactionMechanismGenerator/RMG-Py' }}` now the "reference" job is only the official main ubuntu, all other jobs (including forks) are "dynamic". 

I'm get to see if forks can successfully download the reference results (or if they need a github access token)

Reply:
Yep you're right. I missed the `!` in the statement.